ui/handle_input.go

```diff
@@ -418,3 +452,31 @@ func (win *Window) sortTransactions() {
 		return frontTime.Before(backTime)
 	})
 }
+
+func (win *Window) updateToTransactionDetailsBtns() {
```

Comment:
you can make the "Btn" in the method name fully spelt as in "updateToTransactionDetailsBtns"